config/uniconfig/frinx/uniconfig/config/lighty-uniconfig-config.json

```diff
@@ -230,7 +230,7 @@
     "tls": {
         "enabledTls": true,
         "enabledClientAuthentication": false,
-        "keystorePath": "config/.keystore",
+        "keystorePath": "/run/secrets/frinx_uniconfig_keystore",
```

Comment:
you can keep this path by assigning a "target" path for this secret in docker compose